speechbrain/lobes/models/huggingface_transformers/discrete_ssl.py

```diff
@@ -56,38 +60,37 @@ class DiscreteSSL(nn.Module):
     >>> ssl_layer_num = [7,23]
     >>> deduplicate =[False, True]
     >>> bpe_tokenizers=[None, None]
-    >>> kmeans_repo_id = "speechbrain/SSL_Quantization"
-    >>> kmeans_dataset = "LJSpeech"
+    >>> vocoder_repo_id = "speechbrain/hifigan-wavlm-k1000-LibriTTS"
+    >>> kmeans_dataset = "LibriSpeech"
     >>> num_clusters = 1000
     >>> ssl_model = HuBERT(model_hub, save_path,output_all_hiddens=True)
-    >>> model = DiscreteSSL(save_path, ssl_model, kmeans_repo_id=kmeans_repo_id, kmeans_dataset=kmeans_dataset,num_clusters=num_clusters)
-    >>> tokens, embs ,pr_tokens= model(inputs,SSL_layers=ssl_layer_num, deduplicates=deduplicate, bpe_tokenizers=bpe_tokenizers)
+    >>> model = DiscreteSSL(save_path, ssl_model, vocoder_repo_id=vocoder_repo_id, kmeans_dataset=kmeans_dataset,num_clusters=num_clusters)
+    >>> tokens = model.encode(inputs,SSL_layers=ssl_layer_num, deduplicates=deduplicate, bpe_tokenizers=bpe_tokenizers)
     >>> print(tokens.shape)
     torch.Size([3, 6, 2])
-    >>> print(embs.shape)
-    torch.Size([3, 6, 2, 1024])
-    >>> print(pr_tokens.shape)
-    torch.Size([3, 6, 2])
     """
 
     def __init__(
         self,
         save_path,
         ssl_model,
         kmeans_dataset,
-        kmeans_repo_id="speechbrain/SSL_Quantization",
+        vocoder_repo_id="speechbrain/hifigan-wavlm-k1000-LibriTTS",
```

Comment:
It would be good to list in the docstring the available options for `ssl_model`, `kmeans_dataset`, `vocoder_repo_id`, `num_cluster`, `layer_num`